lib/actions/lookup.js

```diff
@@ -12,7 +11,7 @@ const SALESFORCE_API_VERSION = '39.0';
 module.exports.getMetaModel = async function getMetaModel(configuration) {
   configuration.metaType = 'lookup';
   const metaLoader = new MetaLoader(configuration);
-  return await metaLoader.loadMetadata();
```

Comment:
Why the `await` has been removed here? Should we return back async behavior?

Reply:
await is excessive here, async functions always return Promise so await is just add one more Promise wrapper.